std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/DoubleStorage.java

```diff
@@ -415,4 +417,12 @@ protected Long computeItem(int idx) {
   public BitSet getIsNothingMap() {
     return isNothing;
   }
+
+  @Override
+  public double get(long index) throws ValueIsNothingException {
+    if (isNothing(index)) {
+      throw new ValueIsNothingException(index);
+    }
+    return getItem((int) index);
```

Comment:
I'd prefer `Math.toIntExact` here - if someone gives us too large of a n index, we want to throw an exception, not get it modulo truncated.